models/app.models.js

```diff
@@ -43,4 +43,18 @@ exports.patchReviewVotesById = (review_id, inc_votes) => {
         }
         return review
     })
-}
\ No newline at end of file
+}
+
+exports.selectCommentsByReviewId = (review_id) => {
+    return db.query(`
+    SELECT comments.* 
+    FROM comments
+    WHERE review_id = $1
+    ORDER BY created_at DESC`, [review_id]).then(({ rows: comments }) => {
+        if (comments.length === 0) {
```

Comment:
Does this logic distinguish between an article with no comments and an article that does not exist?